index.html

```diff
@@ -1674,7 +1674,9 @@ <h3>Validity Period</h3>
 <a>credential</a> becomes valid, which could be a date and time in the future.
 Note that this value represents the earliest point in time
 at which the information associated with the <code>credentialSubject</code>
-<a>property</a> becomes valid.
+<a>property</a> becomes valid. If a `validUntil` value also exists, the
+`validFrom` value MUST be temporally the same or earlier than the `validUntil`
```

Comment:
I wonder if `temporally the same` could be confusing (both to readers and implementers) since it allows for a different format value that complies with the format restrictions, if the time value is the same, I think...?

Like -- I could write a date as `2002-04-20T05:55:55Z` or `2020-04-20T05:55:55+00:00` and they're temporally the same, though doing this may increase risk an implementation breaks for some reason. Probably not a big concern.

If you agree there is a potential risk firming up the language to "the same" would remove this

Reply:
>  it allows for a different format value that complies with the format restrictions, if the time value is the same

Yes, exactly, we want that; that is good. :)

I believe the whole point of saying "temporally the same" is to indicate that the comparison is happening in value space, not lexical space. If an implementation breaks in value space when two lexical values should be "temporally the same", then that's an implementation bug in the time value processing code and we tend to not write advice for potential implementation bugs at that layer (unless they're common, which they're not, IMHO).

We could say "the same time value (note that different lexical time strings can represent the same temporal value)"... but that's a bit of a mouthful and raises even more questions that would require further explanation about why comparing time values is a difficult and complicated thing (when you get down into the time processing library code).

All that to say, I think we're good here, unless I missed something and you can provide some concrete text that would address your concern?

Reply:
agree on the small risk, which is an impl bug, but my concern is non blocking